lapis/src/test/kotlin/org/genspectrum/lapis/model/FastaHeaderTemplateTest.kt

```diff
@@ -0,0 +1,257 @@
+package org.genspectrum.lapis.model
+
+import com.fasterxml.jackson.databind.node.BooleanNode
+import com.fasterxml.jackson.databind.node.DoubleNode
+import com.fasterxml.jackson.databind.node.IntNode
+import com.fasterxml.jackson.databind.node.NullNode
+import com.fasterxml.jackson.databind.node.TextNode
+import org.genspectrum.lapis.config.DatabaseConfig
+import org.genspectrum.lapis.config.DatabaseMetadata
+import org.genspectrum.lapis.config.DatabaseSchema
+import org.genspectrum.lapis.config.MetadataType
+import org.genspectrum.lapis.config.OpennessLevel
+import org.genspectrum.lapis.controller.BadRequestException
+import org.genspectrum.lapis.request.CaseInsensitiveFieldsCleaner
+import org.hamcrest.CoreMatchers.`is`
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.containsString
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class FastaHeaderTemplateTest {
+    private val fastaHeaderTemplateParser = FastaHeaderTemplateParser(
```

Comment:
This looks like a copy of the template parser in `lapis/src/test/kotlin/org/genspectrum/lapis/model/SiloQueryModelTest.kt` can we reuse?

Reply:
Hm, we could. But I think I'd rather leave them separate.

One reason why you usually try to reuse stuff is so that you don't have to do the same changes in multiple places. but those are separate tests, they can have different requirements.

But I think it makes sense to a least extract something for the verbose database config, I'll do that.